### PR TITLE
names.md

### PR DESCRIPTION
**Pull request by jenner-luo**
Add Chinese translation of names.md

### CODE REVIEW

**Comment by xichengliudui** [comment]
@jenner-luo  Thank you for your initial contribution, but the Chinese translator is working according to release1.12. Can you work in release1.12?
Please sign the cla agreement (it only needs to be signed once).

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+reviewers:
+- mikedanese
+- thockin
+title: 名称(Names)
```
Comment (chenrui333) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]: 标题就不用英文了。
```suggestion
title: 名称
```

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Kubernetes REST API中的所有对象都由名称和UID明确标识。
```
Comment (chenrui333) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
Kubernetes REST API 中的所有对象都由名称和 UID 明确标识。
```

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+对于非惟一的用户提供的属性，Kubernetes提供[标签](/docs/user-guide/labels) 和[注释](/docs/concepts/overview/working-with-objects/annotations/)。
```
Comment (chenrui333) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
对于非惟一的用户提供的属性，Kubernetes 提供[标签](/docs/user-guide/labels) 和[注释](/docs/concepts/overview/working-with-objects/annotations/)。 
```

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+weight: 20
+---
+
+{{% capture overview %}}
+
+Kubernetes REST API中的所有对象都由名称和UID明确标识。
+
+对于非惟一的用户提供的属性，Kubernetes提供[标签](/docs/user-guide/labels) 和[注释](/docs/concepts/overview/working-with-objects/annotations/)。 
+
+有关名称和uid的精确语法规则，请参阅[标识符设计文档](https://git.k8s.io/community/contributors/design-proposals/architecture/identifiers.md)。
```
Comment (chenrui333) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]: 需要加上英文原文，这样才好校对。
```suggestion
有关名称和 UID 的精确语法规则，请参阅[标识符设计文档](https://git.k8s.io/community/contributors/design-proposals/architecture/identifiers.md)。 
```

File: content/zh/docs/concepts/overview/working-with-objects/names.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## 名称
+
+{{< glossary_definition term_id="name" length="all" >}}
+
+按照惯例，Kubernetes资源的名称最大长度应为253个字符，由小写字母数字字符“-”和“.”组成，但某些资源有更具体的限制。
```
Comment (chenrui333) [inline content/zh/docs/concepts/overview/working-with-objects/names.md]:
```suggestion
按照惯例，Kubernetes 资源的名称最大长度应为 253 个字符，由小写字母数字字符“-”和“.”组成，但某些资源有更具体的限制。
```

**Comment by chenrui333** [comment]
@jenner-luo ping

**Comment by tengqm** [comment]
Feel free to reopen when CLA is signed. Thanks.

**Pull request closed without merging**